index.html

```diff
@@ -9,7 +9,7 @@
     <meta name="description" content="Avocado - Automated Testing Framework">
     <meta name="author" content="">
 
-    <title>Avocado - Testing made easier</title>
+    <title>Avocado Testing Framework</title>
```

Comment:
I don't want to keep them all in sync. I think it's positive to have "Avocado Testing Framework", "Avocado - Automated Testing Framework" and "Avocado Framework" all there. It doesn't bother me at all and I don't think it hurts.

What bothers me is the headline "Testing Made Easier", which is the only thing I'm changing here.